src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,137 @@
+              }),
+            },
+          }}
+        >
+          <Input
+            overrides={{
+              Root: {
+                props: {
+                  'data-testid': 'username',
+                },
+              },
+              Input: {
+                style: ({ $theme }) => ({
+                  backgroundColor: `${$theme.colors.inputBackground}`,
+                  color: `${$theme.colors.fontColor}`,
+                  caretColor: `${$theme.colors.fontColor}`,
+                }),
+              },
+            }}
+            name="username"
+            placeholder="Email address"
+            value={formik.values.username}
+            onChange={formik.handleChange}
+            onBlur={formik.handleBlur}
+            autoComplete="off"
+          />
+        </FormControl>
+      </Block>
+      <Block>
+        <FormControl
+          label={() => 'Password'}
+          error={formik.errors.password}
+          overrides={{
+            Label: {
+              style: ({ $theme }) => ({
+                color: `${$theme.colors.fontColor}`,
+              }),
+            },
+          }}
+        >
+          <Input
+            overrides={{
+              Root: {
+                props: {
+                  'data-testid': 'password',
+                },
+              },
+              Input: {
+                style: ({ $theme }) => ({
+                  backgroundColor: `${$theme.colors.inputBackground}`,
+                  color: `${$theme.colors.fontColor}`,
+                  caretColor: `${$theme.colors.fontColor}`,
+                }),
+              },
+            }}
+            name="password"
+            placeholder="Password"
+            value={formik.values.password}
+            onChange={formik.handleChange}
+            onBlur={formik.handleBlur}
+            autoComplete="off"
+            type="password"
+          />
+        </FormControl>
+      </Block>
+      <Block style={{ marginTop: '1rem', marginBottom: '1rem' }}>
+        <Checkbox
+          name="rememberMe"
+          checked={formik.values.rememberMe}
+          onChange={formik.handleChange}
+          labelPlacement={LABEL_PLACEMENT.right}
+          overrides={{
+            Checkmark: {
+              style: ({ $theme }) => ({
+                backgroundColor: `${$theme.colors.inputBackground}`,
+                border: `1px solid ${$theme.border.borderColor}`,
+                borderRadius: '0.25em',
+              }),
+            },
+            Label: {
+              style: ({ $theme }) => ({
+                color: `${$theme.colors.fontColor}`,
+              }),
+            },
+          }}
+        >
+          Remember me
+        </Checkbox>
+      </Block>
+      <Block>
+        <Button
+          size={SIZE.default}
+          overrides={{
+            BaseButton: {
+              style: () => ({
+                width: '100%',
+              }),
+            },
+          }}
```

Comment:
For now, let's remove this and center the button.

Reply:
Used a boolean prop to handle this